### DISCUSSION TITLE
preql Jupyter cell magic

### DISCUSSION THREAD

**Discussion by dimitri-b** [erezsh/Preql#35] [Ideas]
I'm not sure if it is possible, but It would be great to have a cell magic for `preql`, with the auto-completion and syntax highlighting in notebooks with Python kernel.
This way one can use `preql` cell to build a table and Python cells to chart it or further process it if more complex processing is needed.

**Comment by erezsh** [comment]
I'm not sure I understand what you mean by it. Can you give an example of the usage you envision?

**Comment by dimitri-b** [comment]
Here is an example of what it might look like inside a single notebook, either on Python or Preql kernel - users can mix and match Python and Preql cells within one notebook using cell magic commands.

<img width="1058" alt="2021-08-26_10-05-58" src="https://user-images.githubusercontent.com/4684795/130880414-541b6d90-ec7e-4140-a7a8-3bde99d8a35f.png">
